doc/ref/files.xml

```diff
@@ -100,6 +96,24 @@ for more information how to do this.
 </Section>
 
 
+<!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
+<Section Label="GAP Package Directories">
+<Heading>GAP Package Directories</Heading>
+<Index Key="GAPInfo.PackageDirectories"><C>GAPInfo.PackageDirectories</C></Index>
+
+When &GAP; is started it determines a list of directories potentially
+containing packages called the <E>&GAP; package directories</E>.
+In a running &GAP; session this list can be found in <C>GAPInfo.PackageDirectories</C>.
+<P/>
+Every subdirectory <F>pkg/</F> in a &GAP; root directory is automatically
+added to this list.
+<P/>
+&GAP; looks for available packages by examining each of the directories in
+<C>GAPInfo.PackageDirectories</C>.
+
```

Comment:
Perhaps this section could point out (via a `<Ref`) that one can use `ExtendPackageDirectories` to add more, and also mention the `--packagedirs` option?

And also `run.xml` could be amended to document this new option (see PR #5878 for a hint how to do that).